app/services/symphony_client.rb

```diff
@@ -166,6 +166,8 @@ def error_code(response)
       return if response.status.ok?
 
       JSON.parse(response.body).dig('messageList')[0].dig('code')
```

Comment:
Not part of this PR, but what do you think of:

```suggestion
      JSON.parse(response.body).dig('messageList').first.dig('code')
```

I think it's easier to read, more ruby-ish